docs/source/en/model_doc/qwen2_vl.md

```diff
@@ -217,28 +217,30 @@ print(output_text)
 
 ### Usage Tips
 
-#### Image Resolution for performance boost
+#### Image Resolution trade-off
 
-The model supports a wide range of resolution inputs. By default, it uses the native resolution for input, but higher resolutions can enhance performance at the cost of more computation. Users can set the minimum and maximum number of pixels to achieve an optimal configuration for their needs.
```

Comment:
The previous comment was better. Perhaps we can introduce the example below after the initial introduction to something like `"In the case of limited GPU RAM, one can reduce the resolution, as follows:"`

Reply:
Done